tensorflow-framework/src/main/java/org/tensorflow/framework/metrics/impl/MetricVariable.java

```diff
@@ -0,0 +1,125 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=======================================================================*/
+package org.tensorflow.framework.metrics.impl;
+
+import org.tensorflow.Operand;
+import org.tensorflow.framework.initializers.Glorot;
+import org.tensorflow.framework.initializers.Initializer;
+import org.tensorflow.framework.initializers.VarianceScaling;
+import org.tensorflow.framework.initializers.Zeros;
+import org.tensorflow.op.Ops;
+import org.tensorflow.op.core.Variable;
+import org.tensorflow.types.family.TFloating;
+import org.tensorflow.types.family.TIntegral;
+import org.tensorflow.types.family.TNumber;
+
+/**
+ * Helper class that holds a metric variable
+ *
+ * @param <U> the data type of the variable
+ */
+public class MetricVariable<U extends TNumber> {
```

Comment:
You're right, I was using it, but  stopped using it when I changed the way `Variables` are kept. They are now attributes of the  class rather than stored in a Map as before. We could delete it now.

Reply:
👍 done (by deleting the file)